Xamarin.Forms.ControlGallery.iOS/Main.cs

```diff
@@ -8,7 +8,7 @@ public class Application
 	{
 		static void Main(string[] args)
 		{
-			UIApplication.Main(args, null, "AppDelegate");
+			UIApplication.Main(args, "Application", "AppDelegate");
```

Comment:
Can we use this instead
```
UIApplication.Main(args, typeof(CustomApplication), typeof(AppDelegate));
```

Just makes me feel better than having strings. Then we can also remove the Register on CustomApplication

Reply:
done)